tools/release.sh

```diff
@@ -14,6 +16,20 @@ pwd
 
 make HELIO_RELEASE=y release
 
-build-opt/dragonfly --version
+if ! [ -f ${APP_PATH} ]; then
+   echo "ERROR"
+   echo "Failed to generate new dragonfly binary."
+   exit 1
+fi
+
+${APP_PATH} --version
+
+if readelf -a ${APP_PATH} | grep GLIBC_PRIVATE >/dev/null 2>&1 ; then
+   echo "ERROR"
+   echo "The generated binary contain invalid GLIBC version entries."
+   echo "This will failed to run in same cases. Please fix this issue"
+   exit 1
+fi
 
 make package
+exit $?
```

Comment:
`set -e` already takes care of that, no need for line 35

Reply:
ok